io.catenax.sealant.sealant_pass/1.0.0/SealantPass.ttl

```diff
@@ -0,0 +1,648 @@
+:declarationOfConformity a bamm:Property;
+    bamm:characteristic :LocateCharacteristic;
+    bamm:preferredName "declaration of conformity"@en;
+    bamm:description "Declaration of conformity (CE) is describing a regulatory requirement from the EU."@en.
+:removalProcedure a bamm:Property;
+    bamm:characteristic :LocateCharacteristic;
+    bamm:preferredName "removal procedure"@en;
+    bamm:description "The documents explain the removal and possible recycling options of the product."@en.
+:dueDiligence a bamm:Property;
+    bamm:characteristic :LocateCharacteristic;
+    bamm:preferredName "due diligence"@en;
+    bamm:description "Report on Supply Chain Due Diligence policies. Link to document or website on its supply chain due diligence policies."@en.
+:technicalDocumentation a bamm:Property;
+    bamm:characteristic :LocateCharacteristic;
+    bamm:preferredName "technical documentation"@en;
+    bamm:description "Technical documentation refers to documents that explain the use, functionality, creation, or architecture of the product."@en.
+:fullSDS a bamm:Property;
+    bamm:characteristic :SpecificsSDSCharacteristic;
+    bamm:preferredName "full SDS"@en;
+    bamm:description "A link to the full SDS of the material. A link to a SDS might be provided for several countries and possibly multiple languages for each country. The table includes: Version, Countryname, language of the document and the link to the document. The documents should preferably be provided as PDF or Word document."@en.
+:documentTitle a bamm:Property;
+    bamm:characteristic :DocumentTitleTrait;
+    bamm:preferredName "document title"@en;
+    bamm:description "The header or title of the document"@en;
+    bamm:exampleValue "Cobald Mine A".
+:documentLocation a bamm:Property;
+    bamm:characteristic bamm-c:Text;
+    bamm:preferredName "document location"@en;
+    bamm:description "Location or link of and to the document. "@en;
+    bamm:exampleValue "ftp://www.company.com/MyCompliance/Compliant02.PDF".
+:sustainabilityReport a bamm:Property;
+    bamm:characteristic :LocateCharacteristic;
+    bamm:preferredName "sustainability report"@en;
+    bamm:description "Link to public website or Document. Reporting in accordance with the new European Sustainability Reporting Standards (ESRS) including the new ESG reporting standards."@en.
+:DocumentTitleTrait a bamm-c:Trait;
+    bamm-c:baseCharacteristic bamm-c:Text;
+    bamm-c:constraint :MaxTitleCharacterConstraint.
+:MaxTitleCharacterConstraint a bamm-c:LengthConstraint;
+    bamm:preferredName "max title character"@en;
+    bamm:description "Maximum title characters set to 60."@en;
+    bamm-c:maxValue "60"^^xsd:nonNegativeInteger;
+    bamm-c:minValue "0"^^xsd:nonNegativeInteger.
+:SpecificsSDSCharacteristic a bamm-c:Set;
+    bamm:dataType :SpecificsSDSEntity;
+    bamm:preferredName "specifics SDS"@en;
```

Comment:
All occurrences of SDS consider adding an expansion of the acronym to the description

Reply:
done